paper-year-list.html

```diff
@@ -168,35 +171,33 @@
           this.date = new Date(this.date.setFullYear(selected));
         }
         this._selectYearInList(selected);
-      },
+      }
       /**
        * Select the given year in the years list.
        */
-      _selectYearInList: function(year) {
+      _selectYearInList(year) {
         var yearIdx = year - this.min;
-        this.$.yearList.selectItem(yearIdx);
-      },
+        this.$.yearList.selectIndex(yearIdx);
+      }
       /**
        * Update 'selected' attribute and select in iron-list
        * from a tapped item's event in the years list.
        */
-      _tappedYearHandler: function(e) {
-        var yearItem = e.model.__data__.item;
+      _tappedYearHandler(e) {
+        var yearItem = e.model.__data.item;
```

Comment:
We shouldn't be messing with __data on the model at all, `e.model.item` *should* have the same data, if I remember correctly.

Reply:
Fixed i probably found the data first in __data